src/_pyinstaller_hooks_contrib/hooks/stdhooks/hook-ffpyplayer.py

```diff
@@ -0,0 +1,19 @@
+# ------------------------------------------------------------------
+# Copyright (c) 2021 PyInstaller Development Team.
+#
+# This file is distributed under the terms of the GNU General Public
+# License (version 2.0 or later).
+#
+# The full license is available in LICENSE.GPL.txt, distributed with
+# this software.
+#
+# SPDX-License-Identifier: GPL-2.0-or-later
+# ------------------------------------------------------------------
+
+from PyInstaller.utils.hooks import get_module_attribute, collect_submodules
+
+hiddenimports = collect_submodules("ffpyplayer")
+binaries = []
+# ffpyplayer has an internal variable tells us where the libraries it was using
+for bin in get_module_attribute("ffpyplayer", "dep_bins"):
```

Comment:
Ah sorry, looks like my previous comment regarding `get_module_attribute` was taken as request for change. The problem is that with PyInstaller 4.x, get_module_attribute returns string-ified version of attribute value, i.e.:

```
>>> from PyInstaller.utils.hooks import get_module_attribute
>>> get_module_attribute('ffpyplayer', 'dep_bins')
'[]'
```

So this needs to be `eval`d to get the list.

In PyInstaller 5.x we switched to new process isolation framework that returns proper native types, so:

```
>>> from PyInstaller.utils.hooks import get_module_attribute
>>> get_module_attribute('ffpyplayer', 'dep_bins')
[]
```

(i.e., the list is returned and can be used directly).

So this would require different handling based on PyInstaller version (that's why my previous comment was meant more as a side note, and mentioned the hassle of handling both).